build.ps1

```diff
@@ -5,29 +5,30 @@ Used to start the build of a PowerShell Module
 This script will install dependencies using PSDepend module and start the build tasks using InvokeBuild module
 .NOTES
 Change History
--1.0 | 2019/06/17 | Francois-Xavier Cat
-    Initial version
+-1.0 | 2019/06/17 | Francois-Xavier Cat (lazywinadmin)
```

Comment:
These seems to be the changes i did yesterday

Reply:
The issue I found was that you cannot put any other links within the `.NOTES` section of any given comment block.  More specifically, the `.NOTES` section has to contain the following URL, or else the build will fail... (this URL is required: `https://github.com/lazywinadmin/ADSIPS`)  In my notes, I added the URL for each issue I was resolving, this caused the build to fail.

<img width="1036" alt="Screen Shot 2019-06-23 at 1 00 38 PM" src="https://user-images.githubusercontent.com/21092343/59980140-34dd8f80-95b7-11e9-825a-daaf7975eba3.png">

Reply:
Yessir - I had to copy that entire file into my fork of the repo, or else I was getting merge conflicts.  I did not change anything - just copied the file to my fork to resolve the merge conflicts.

Reply:
no problem, thanks again for your help

Reply:
No problem at all. Thanks for allowing me to contribute!